test/Microsoft.OpenAPI.OData.Reader.Tests/Edm/EdmModelExtensionsTests.cs

```diff
@@ -55,5 +59,23 @@ public void FindAllBaseTypesReturnsCorrect()
             Assert.Equal(2, baseTypes.Count());
             Assert.Equal(new[] { subEntityType, baseEntityType }, baseTypes);
         }
+
+
+        [Fact]
+        public void GetAllElementsReturnsElementsFromAllModels()
+        {
+            // Arrange
+            IEdmModel model = EdmModelHelper.InheritanceEdmModelAcrossReferences;
+
+            // Act
+            var elements = model.GetAllElements();
+
+            // Assert
+            Assert.Collection(elements, 
+                e => Assert.Equal("Customer", e.Name),
+                e => Assert.Equal("Default", e.Name),
+            e => Assert.Equal("CustomerBase", e.Name));
```

Comment:
alignment and remove the unused line at the end

Reply:
Done.